airqo-platform/AirQo-frontend
# [Netmanager] Landing Page Map Fixes & Logout bug

#### Summary of Changes (What does this PR do?)

- Updated auth token when the user logs out
- Maintained map config when the user logs out(ie keep pollutant value, map style, map mode)
- Update the top navbar icons when the user logs out
- Resolved brown nodes bug

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- Login to the platform. Go to the map page and try to logout. It should transition to the landing page seamlessly.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [AN-227](https://airqoteam.atlassian.net/browse/AN-277)

[AN-227]: https://airqoteam.atlassian.net/browse/AN-227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
